squid/src/main.ts

```diff
@@ -29,39 +24,33 @@ import { Env } from './util/Env'
 import { getAccountId } from './util/getAccountId'
 import { getProxyAccountIByDelegatorIds } from './util/getProxyAccountIByDelegatorIds'
 
-export const dataEvent = {
-  data: {
-    event: {
-      args: true
-    }
-  }
-} as const
-
-export const dataCall = {
-  data: {
-    call: {
-      args: true,
-      origin: true
-    }
-  }
-} as const
-
 const supportedMultisigCalls = [
   'Multisig.as_multi',
   'Multisig.approve_as_multi',
   'Multisig.cancel_as_multi',
   'Multisig.as_multi_threshold_1'
 ]
 export const env = new Env().getEnv()
-const archiveUrl =
-  env.archiveUrl ||
-  lookupArchive(env.archiveName as KnownArchivesSubstrate, {
-    release: 'FireSquid',
-    genesis: env.genesis,
-    type: 'Substrate'
-  })
+const archiveUrl = env.archiveName
+  ? lookupArchive(env.archiveName as KnownArchivesSubstrate, {
+      release: 'ArrowSquid',
+      type: 'Substrate'
+    })
+  : undefined
 const chainId = env.chainId
 
+export const fields = {
+  call: {
+    args: true,
+    origin: true,
+    success: true
+  },
+  event: {
+    args: true
+  },
```

Comment:
Not every call will have an event data, For example for kill proxy event has no event

Reply:
This is about the structure of events. if it's an event, I need the args. 
https://docs.subsquid.io/substrate-indexing/setup/field-selection/#set-fields

`proxy.killProxy` is a call, it'll have `args`, `origin`, `success`, which is what we use.